databus-client2/src/main/java/com/bazaarvoice/emodb/databus/client2/discovery/EmoServiceDiscovery.java

```diff
@@ -0,0 +1,197 @@
+package com.bazaarvoice.emodb.databus.client2.discovery;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.common.base.Throwables;
+import com.google.common.collect.Lists;
+import com.google.common.io.Closeables;
+import com.google.common.util.concurrent.AbstractService;
+import org.apache.curator.framework.CuratorFramework;
+import org.apache.curator.framework.recipes.cache.ChildData;
+import org.apache.curator.framework.recipes.cache.PathChildrenCache;
+import org.apache.curator.utils.ZKPaths;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.core.UriBuilder;
+import java.io.IOException;
+import java.io.Serializable;
+import java.net.URI;
+import java.net.UnknownHostException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import static com.bazaarvoice.emodb.databus.client2.Json.JsonUtil.parseJson;
+import static com.google.common.base.Preconditions.checkArgument;
+
+
+/**
+ * Service discovery for EmoDB.  This uses the same logic as in the EmoDB Java client but without the incompatible
+ * dependencies introduced by the current client.
+ */
+public abstract class EmoServiceDiscovery extends AbstractService {
+
+    private static final Logger _log = LoggerFactory.getLogger(EmoServiceDiscovery.class);
+
+    private final String _zookeeperConnectionString;
+    private final String _zookeeperNamespace;
```

Comment:
field is unused, please double check

Reply:
it is being using in sub class DatabusDiscovery.java https://github.com/bazaarvoice/emodb/pull/391/files#diff-e3c7718a6eddf7ed4ce4f6f380d1c906adf27cef556eef7ab9c61f84f61d9bd5R56-R57

Reply:
as was correctly mentioned https://github.com/bazaarvoice/emodb/pull/391/files#diff-6aa4855ae8d550af1862cddb7157461ef3748db78289fdaba3c9682ab8243132R160 is used, but not this field